bacon/model/StorageCouchBaseDB.swift

```diff
@@ -97,10 +97,11 @@ class StorageCouchBaseDB {
         return folderPath
     }
 
-    private func createMutableDocument(from transaction: Transaction) throws -> MutableDocument {
+    private func createMutableDocument(from transaction: Transaction, id: String? = nil) throws -> MutableDocument {
         do {
             let transactionData = try transaction.asDictionary()
-            let transactionDocument = MutableDocument(data: transactionData)
+            // If id is nil, a random UID will be assigned to the document
```

Comment:
I thought waikay mentioned that the ID should not be generated by the DB?

edit: Oh, just realized this is a private method. The ID is for internal use only, right?

Reply:
ya